go.mod

```diff
@@ -3,47 +3,17 @@ module github.com/AbsaOSS/k8gb
 go 1.13
```

Comment:
Can we try to migrate to go1.15? OR in separate issue ?

Reply:
@kuritka I would better not mix it , also `go 1.13` spec was created by framework generator so don't really want to override it given complexity of this PR